Robust.Client/Robust.Client.csproj

```diff
@@ -40,7 +40,9 @@
       <LogicalName>Robust.Client.Graphics.RSI.RSISchema.json</LogicalName>
     </EmbeddedResource>
 
-    <EmbeddedResource Include="Graphics\Clyde\Shaders\*" />
+    <Content Include="Graphics\Clyde\Shaders\*">
```

Comment:
I'd prefer if these were put back into `Resources/` instead of using `CopyToOutputDirectory`.